internal/2022-q4-ashfrancis.md

```diff
@@ -0,0 +1,39 @@
+
+### Background 
+
+The [Incubator rules doc (v3)](https://github.com/dashincubator/dash-incubator-rules/blob/042fde0eea2340a2a8c9fa3483ad8f764d94fe31/rules.md) directs individual strategists to submit internal budget proposals to the Lead Strategist.
+
+From the Rules document:
+
+> 5.1.2 Strategist Proposals #
+> Each Strategist submits quarterly proposals to the Lead Strategist to increase their budgets. The content and purpose of this proposal is similar to the Lead Strategist’s Network Proposal. It should report project updates, value propositions, budget spending, quarterly priorities, and anything else the strategist wants to commmunicate in order to help justify the value delivered to the network for the funding received and requested.
+
+### Project Updates & Budget Spending
+
+In Q3 2022 I funded the following Bounties:
+
+* [Jembe Dapp](https://trello.com/c/Kzn8JX12/12-jembe-dapp) - A decentralized twitter analog built on Dash Platform. There was an outstanding amount of Design work that has now been finalized and this project paused pending a more stable platform environment.
+* [Rust Dash RPC Client](https://trello.com/c/xp5NWpdO/211-rust-dash-rpc-client) - Porting a Go based RPC client for L1 to Rust. This work is on-going.  
+
+My requested budget for the period was 900 Dash, of which 420 Dash was distributed in rewards. There is currently around 200 Dash in outstanding tasks awaiting work to be delivered and task to be claimed and awarded. Remaining funds will be allocated towards a buffer. 
+
+My Q3 proposal also included the following bounties:
+
+* [THORChain](https://trello.com/c/CDKTVi9x/152-thorchain-integration) - This work is completed from a development side outside of maintenance. 
+* [Governance Portal](https://trello.com/c/c36t7QSM/141-governance-portal) - Work hasn't yet resumed on this project
+* THORChain Lite Swapping Interface - This work has been postponed until after the Summoners Game 
+
+### Quarterly Priorities & Value Propositions
+
+During the upcoming quarter I plan to focus on the RUST RPC Client, Governance Portal, THORChain PR maintenance and the following bounties:
+
+* [DashPay Extension](https://trello.com/c/8Hy6K7fE) - An extension that acts as the gateway to DAPP usage and includes chat and other novel features. 
+* [Summoners Dash Platform Game](https://trello.com/c/mRKNrEzb/215-summoners-dash-platform-game) - A turn based strategy game that allows bets by players on outcome with data stored on platform and use of L1 for transactions. 
+* A DCG Originated Dash Extension (Bounty Card TBC) - Taking an extension for L1 originally created by DCG and finishing off various features to make it production ready.  
+
+### Budget Request
+
+I am requesting 400 DASH/month for 3 months from the Incubator general budget. This will cover the existing and new work as above based on cost forecasts. Any remaining funds will be applied to a buffer to ensure development continuity. 
```

Comment:
Please remove the "from the Incubator general budget" part, since the funds come from my strategist reserve, not the general reserve.